content/kb/restoring-data-from-an-orphaned-replica-directory.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Restoring Data from an Orphaned Replica Directory"
+authors:
+- "Sushant Gaurav"
+draft: false
+date: 2025-09-02
+versions:
+- All
+categories:
+- "Data Recovery"
+- "Troubleshooting"
+---
+
+## Applicable versions
+
+Longhorn v1.10.x versions.
+
+## Overview
+
+This guide explains how to restore data from a **replica directory** that is no longer being managed by Longhorn. This situation is referred to as **orphaned data** and can occur if the Longhorn system loses track of the replica, or if a volume is accidentally deleted without properly detaching the replica directories.
+
+Although the Longhorn UI no longer manages this replica, the raw volume data remains on the disk. By following the steps below, you can access and recover your data.
+
+## Before you begin
+
+This process involves running commands on the node where the orphaned replica is located. You will need `SSH` access to that node.
+
+Additionally, to avoid potential data corruption, you must verify that the replica is **not in use** by any active processes. Use the `lsof` command to check for any open files within the directory.
```

Comment:
```suggestion
To avoid potential data corruption, you must verify that the replica is not in use by Longhorn or any other active processes. For example, you can use commands such as `lsof` or `fuser` to check whether any processes are accessing files within the replica directory.
```